### PR TITLE
links_updated

### PR DESCRIPTION
**Pull request by waqaskanju**
links updated text added
src update alt added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,164 +1,154 @@
...
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
```
Comment (waqaskanju) [inline index.html:7]: I would be better to you a proper title

File: index.html [hunk 2]
```diff
@@ -1,164 +1,154 @@
...
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
 
+    <style>
```
Comment (waqaskanju) [inline index.html:18]: It is better to use separate file for css. An external style sheet.

File: index.html [hunk 3]
```diff
@@ -1,164 +1,154 @@
...
-  }
-}
-
-
-  
- </style>
-<script>
-
-
-</script>
```
Comment (waqaskanju) [inline index.html:66]: I think script tag should be at the end of document. to increase reload time.  Kindly do not leave. empty code in production

File: index.html [hunk 4]
```diff
@@ -1,164 +1,154 @@
...
-        <div class="container-fluid">
-          <div class="navbar-header">
-            <a class="navbar-brand" href="#">Waqas & Alier Team</a>
-          </div>
-          <ul class="nav navbar-nav ">
-            <li class="active"><a href="#">Home</a></li>
-            <li><a href="#welcome-section">About</a></li>
-            <li><a href="#">Work</a></li>
-            <li><a href="#">Contact</a></li>
-          </ul>
```
Comment (waqaskanju) [inline index.html:79]: USe Nav tag for Link. It is better for semantics and screen readers.

File: index.html [hunk 5]
```diff
@@ -1,164 +1,154 @@
...
+              ><i class="fa fa-github" style="font-size: 40px">GitHub</i>
+            </a>
+          </div>
+          <div class="col-sm-2">
+            <i class="fa fa-twitter" style="font-size: 40px">Twitter</i>
+          </div>
+          <div class="col-sm-2">
+            <i class="fa fa-at" style="font-size: 40px">Email</i>
           </div>
+          <div class="col-sm-2">
```
Comment (waqaskanju) [inline index.html:138]: Use extranl css instead of inline

File: index.html [hunk 6]
```diff
@@ -1,164 +1,154 @@
...
+            <i class="fa fa-phone" style="font-size: 40px">Call</i>
+          </div>
+        </div>
       </div>
       <div class="row hr">
-          <div class="col-sm-8"> This is our project portfolio with Waqas.</div>
-          <div class="col-sm-4"> Created for Microverse Full Program</div>
-
+        <div class="col-sm-8">This is our project portfolio with Waqas.</div>
+        <div class="col-sm-4">Created for Microverse Full Program</div>
```
Comment (waqaskanju) [inline index.html:145]: It is better not to leave empty line. For no reason

**Review by waqaskanju — COMMENTED** [review]
Great work, But I would like to give you some improvements.

File: index.html [hunk 7]
```diff
@@ -1,164 +1,154 @@
...
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+    <main>
+      <header id="welcome-section">
+        <h1>Hey! I am Alier,</h1>
+        <h3>I am web developer</h3>
+      </header>
+      <h1 class="pad">These are some of our projects</h1>
```
Comment (waqaskanju) [inline index.html:82]: Good work for using html 5 semantic elements.

**Review by koomekevo — APPROVED** [review]
# Congratulations :partying_face: 



Hi @waqaskanju, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

- [ ] Use words such as "Kindly" or "Please" in phrasing your sentences in future, for example, Instead of saying "_I would be better to you a proper title_" you can say _"Kindly use a proper title_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by waqaskanju** [comment]
Thanks for your suggesstions.

**Pull request merged**